src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -238,7 +238,7 @@ const SummaryBar = props => {
             </Row>
           </Col>
 
-          <Col className="col-lg-3 col-12 no-gutters">
+          <Col className="d-flex col-lg-3 col-12 no-gutters">
```

Comment:
I am not seeing any css for class d-flex so why its added @angelinatr15?

Overall it's working fine. 

![709](https://user-images.githubusercontent.com/82662768/225728261-a73bbdd6-60d7-49b1-aedd-7240531b7f73.jpg)

Reply:
Hi @PrabhjotSembhi, 

The d-flex class is coming from bootstrap.